src/expr/elim_shadow_converter.h

```diff
@@ -0,0 +1,65 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Andrew Reynolds
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2022 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Implementation of shadow elimination node conversion
+ */
+#include "cvc5_private.h"
+
+#ifndef CVC4__EXPR__ELIM_SHADOW_NODE_CONVERTER_H
+#define CVC4__EXPR__ELIM_SHADOW_NODE_CONVERTER_H
+
+#include "expr/node.h"
+#include "expr/node_converter.h"
+
+namespace cvc5::internal {
+
+/**
+ * This converts a node into one that does not involve shadowing with the
+ * given variables. In particular, if the given vars passed to the constructor
+ * are bound in any quantifier in a subterm of the node to convert, they
+ * are replaced by fresh variables.
+ *
```

Comment:
Clarify that the given variables are the variables bound by the closure given to the constructor. Also, "quantifier" -> "binder"